pkg/query-service/converter/time.go

```diff
@@ -0,0 +1,58 @@
+package converter
+
+type Duration float64
+
+const (
+	Nanosecond           = Second * 1e-9
+	Microsecond          = Second * 1e-6
+	Millisecond          = Second * 1e-3
+	Centisecond          = Second * 1e-2
+	Decisecond           = Second * 1e-1
+	Second      Duration = 1e0
+	Minute               = Second * 60
+	Hour                 = Minute * 60
+	Day                  = Hour * 24
+	Week                 = Day * 7
+)
+
+// durationConverter is an implementation of Converter for durations.
+type durationConverter struct {
+}
+
+func NewDurationConverter() Converter {
+	return &durationConverter{}
+}
+
+func FromTimeUnit(u Unit) Duration {
+	switch u {
+	case "ns":
+		return Nanosecond
+	case "us":
+		return Microsecond
+	case "ms":
+		return Millisecond
+	case "cs":
+		return Centisecond
+	case "ds":
+		return Decisecond
+	case "s":
+		return Second
+	case "m":
+		return Minute
+	case "h":
+		return Hour
+	case "d":
+		return Day
+	case "w":
+		return Week
+	default:
```

Comment:
Any reason to keep second as default

Reply:
For a unit that is not supported, From{X}Unit returns 1, so the value remains unchanged for unknown units. Here `Second` is an alias for 1 = 1e0. I can change it to 1 if that helps.

Reply:
This is fine